ReactiveTask.podspec

```diff
@@ -0,0 +1,26 @@
+Pod::Spec.new do |s|
+  s.name         = "ReactiveTask"
+  # Version goes here and will be used to access the git tag later on, once we have a first release.
+  s.version      = "0.16.0"
```

Comment:
I bumped the version and tagged the latest commit so that the `pod lint` command can find a valid commit with a `.podspec` in it.